cmd/kubeadm/app/master/apiclient.go

```diff
@@ -68,9 +68,12 @@ func CreateClientAndWaitForAPI(file string) (*clientset.Clientset, error) {
 	fmt.Println("[apiclient] Created API client, waiting for the control plane to become ready")
 
 	start := time.Now()
-	wait.PollInfinite(apiCallRetryInterval, func() (bool, error) {
+	err = wait.PollInfinite(apiCallRetryInterval, func() (bool, error) {
 		cs, err := client.ComponentStatuses().List(v1.ListOptions{})
 		if err != nil {
+			if apierrs.IsForbidden(err) {
```

Comment:
since authorizers default to deny, they can forbid requests that are made before they are initialized as the apiserver is coming up. I could maybe see logging forbidden errors, but I wouldn't short-circuit on them